.github/workflows/ci.yml

```diff
@@ -0,0 +1,28 @@
+# This workflow contains all tock-ci, seperated into jobs
+
+name: spot
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  ci-check:
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Rust toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          profile: minimal
+          default: true
+          components: rustfmt, clippy
```

Comment:
clippy not needed just yet but okay

Reply:
Yep, it's also something worth running eventually so I left it in.